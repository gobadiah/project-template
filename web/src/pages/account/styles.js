import { css } from 'emotion';

// eslint-disable-next-line import/prefer-default-export
export const containerClassName = css`
  width: 640px;
`;
