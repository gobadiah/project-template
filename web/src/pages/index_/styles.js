import { css } from 'react-emotion';

export const foo = () => 'bar';

export const helloWorld = css`
  color: green;
`;
