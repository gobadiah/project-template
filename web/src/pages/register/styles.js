import { css } from 'react-emotion';

export const containerClassName = css`
  width: 640px;
  height: 1106px;
`;
