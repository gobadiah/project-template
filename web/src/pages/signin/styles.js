import { css } from 'react-emotion';

// eslint-disable-next-line import/prefer-default-export
export const containerClassName = css`
  width: 640px;
  height: 682px;
`;
