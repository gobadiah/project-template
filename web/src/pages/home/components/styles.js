import styled from 'react-emotion';

import { FlexEnd, WhiteBox } from '~/styles/styled-components';

export const KeyStatsContainer = styled(WhiteBox)`
  height: 630px;
`;

export const KeyStatsTitle = styled(FlexEnd)`
  margin-top: 31px;
`;
