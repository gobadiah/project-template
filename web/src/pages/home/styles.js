import styled from 'react-emotion';

import { Flex, FlexColumn } from '~/styles';

export const Container = styled(Flex)`
  margin: auto;
  width: 969px;
  justify-content: space-between;
`;

export const LeftPanel = styled(FlexColumn)`
  width: 393px;
`;

export const RightPanel = styled(FlexColumn)`
  width: 559px;
`;
