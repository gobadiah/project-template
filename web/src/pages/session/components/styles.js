import styled from 'react-emotion';

import { Flex, FlexCenter, FlexColumn, FlexColumnCenter } from '~/styles';

export const PlayersStatsContainer = styled(FlexColumnCenter)`
`;

export const PlayersStatsLabel = styled(FlexCenter)`
  width: 427px;
  height: 27px;
  margin-bottom: 4px;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  font-stretch: normal;
  line-height: 1.69;
  letter-spacing: normal;
  text-align: center;
  color: var(--dark-blue-grey);
`;

export const PlayersStatsValues = styled(FlexCenter)`
  height: 60px;
`;

export const statBarWidth = 300;

export const StatContainer = styled(FlexColumn)`
  flex-direction: ${props => (props.right ? 'flex-start' : 'flex-end')};
`;

export const StatDisplay = styled('div')`
  font-family: TitilliumWeb;
  font-size: 26px;
  font-weight: normal;
  font-style: normal;
  font-stretch: normal;
  line-height: 1.12;
  letter-spacing: normal;
  text-align: ${props => (props.right ? 'left' : 'right')};
  color: var(--dark-blue-grey);
`;

export const StatColumn = styled(FlexColumnCenter)`
  margin-top: 4px;
`;

export const SessionStatsTable = styled('table')`
  width: 100%;
  margin-top: 16px;
  margin-bottom: 24px;

  thead {
    font-size: 16px;
    font-weight: normal;
    font-style: normal;
    font-stretch: normal;
    line-height: 1.69;
    letter-spacing: normal;
    text-align: left;
    color: var(--bluish-grey);
  }

  tbody {
    font-weight: normal;
    font-style: normal;
    font-stretch: normal;
    line-height: 0.97;
    letter-spacing: normal;
    text-align: left;
    color: var(--dark-blue-grey);
    font-size: 30px;
  }
`;

export const Section = styled('div')`
  text-align: center;
  width: 138px;
  height: 27px;
  font-size: 16px;
  font-weight: bold;
  font-style: normal;
  font-stretch: normal;
  line-height: 1.69;
  letter-spacing: normal;
  text-align: center;
  color: var(--white);
`;

export const SectionsBar = styled(FlexCenter)`
  width: 970px;
  height: 48px;
  background-color: var(--dark-blue-grey);
`;

export const SelectedBar = styled('div')`
  width: 106px;
  height: 4px;
  background-color: var(--apple-green);
  ${props => props.hidden && 'display: none;'}
`;

export const SessionTitleContainer = styled(Flex)`
  align-items: center;
  margin-top: 22px;
  margin-bottom: 13px;
  align-self: flex-start;
`;

export const SectionsContainer = styled(FlexColumn)`
  width: 100%;
`;
