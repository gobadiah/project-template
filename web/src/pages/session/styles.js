import { shape } from 'prop-types';
import React from 'react';
import styled from 'react-emotion';

import { Link } from '~/routes';
import { Flex } from '~/styles';
import { defaultPropTypes } from '~/components';

export const userScoreHeight = 62;

export const containerWidth = 970;

export const UserScores = styled(Flex)`
  margin-top: 31px;
  justify-content: space-between;
  width: 100%;
`;

export const WatchVideo = ({ session }, { t }) => (
  <Link route='video' params={{ id: session.videos[0].id }}>
    <a
      css='
        width: 220px;
        height: 27px;
        margin-bottom: 7px;
        font-size: 16px;
        font-weight: bold;
        font-style: normal;
        font-stretch: normal;
        line-height: 1.69;
        letter-spacing: normal;
        text-align: center;
        color: var(--dark-blue-grey);
      '
    >
      {t('session:Watch the video')}
    </a>
  </Link>
);

WatchVideo.propTypes = {
  session: shape().isRequired,
};

WatchVideo.contextTypes = defaultPropTypes;
