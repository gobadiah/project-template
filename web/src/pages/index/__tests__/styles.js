import styles from '../styles';

describe('Styles', () => {
  describe('Pages', () => {
    describe('index', () => {
      it('should match snapshot', () => {
        expect(styles).toMatchSnapshot();
      });
    });
  });
});
