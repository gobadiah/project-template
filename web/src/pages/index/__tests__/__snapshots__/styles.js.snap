// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles Pages index should match snapshot 1`] = `
Object {
  "helloWorld": "css-bjcoli",
}
`;
